src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Shared.UsingNetSdkDefaults.props

```diff
@@ -0,0 +1,20 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Shared.UsingNetSdkDefaults.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Condition="'$(UsingNETSdkDefaults)' != 'true'">
+    <!-- Skip import of Microsoft.NuGet.props and Microsoft.NuGet.targets -->
+    <SkipImportNuGetProps>true</SkipImportNuGetProps>
+    <SkipImportNuGetBuildTargets>true</SkipImportNuGetBuildTargets>
```

Comment:
I'm not sure if these properties are ever used.  I couldn't find any reference to them.  So delete this file and put the logic in `Microsoft.NET.Sdk.Shared.targets`.  That does reorder the evaluation, but I think it's safe in this case.